### ISSUE TITLE
A question (I'm from Quora)

### ISSUE THREAD

**Issue by hello-doodle** [TheNamesDyl/youth-mentorship#12]
Firstly let me apologise for this intrusion, I have no other way to contact you, I think I ended up in the spam folder of your email. 
Really sorry about this I don't know how else to get across to you

 I was one of your Quora followers. I was disappointed to see your account banned because your technical answers helped me A LOT!

I remember that in one answer you had recommended a book for OOP. I had bookmarked that answer but now its gone with your account :-(

Could you please tell me what book you had recommended for OOP? Or what book you recommend now?

sorry again for doing this. btw I hate dumb Quora mods

**Comment by TheNamesDyl** [comment]
Yeah no problem man, I appreciate the kind words. I will probably start a Quora under a pseudonym just because I liked writing there.

I am not sure which book I recommended, but I personally like http://greenteapress.com/thinkapjava/thinkapjava.pdf. It gave me a good understanding of OOP. It's an object-late kind of textbook, so it goes through procedural programming concepts first, that way readers understand why OOP was developed.
 
Add me on linkedin: https://www.linkedin.com/in/dylanob/. We can keep in touch.
